qa/selenium-ide/web-server/testCases/Vehicles

```diff
@@ -84,3 +84,4 @@
 </tbody></table>
 </body>
 </html>
+<!--covers test TST-18-->
```

Comment:
Would make more sense to me to have this closer to the start of the file. Is there a reason to drop it at the end?
Also, do you think it would make sense to add this as a meta-tag?
```
<meta name="ats:test-case" content="Vehicle whatever test">
<meta name="ats:test-jira-ticket" content="TST-18">
<meta name="ats:test-jira-url" content="https://advancedtelematic...">
```
Or is your HTML somehow auto-generated?